test/compress/classes.js

```diff
@@ -125,7 +125,7 @@ private_methods: {
     }
     expect_exact: "(new class A{static*#f(){yield 3*A.#p}async #g(){for(var a of A.#f())return a*await 2}static get #p(){return 7}get q(){return this.#g()}}).q.then(console.log);"
     expect_stdout: "42"
-    node_version: ">=14"
+    node_version: ">=14.6"
```

Comment:
@kzc this addresses https://github.com/mishoo/UglifyJS/pull/4658#issuecomment-788075261